### PR TITLE
Add test for host policy. Fix edge case

### PR DESCRIPTION
**Pull request by GuanLuo**

### CODE REVIEW

**Comment by GuanLuo** [comment]
Will create follow up PR to update L0_io when we apply the same to other backends.

File: src/backends/backend/triton_backend_manager.cc [hunk 1]
```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2020-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright (c) 2020-2021, NVIDIA CORPORATION & AFFILIATES. All rights
```
Comment (deadeyegoodwin) [inline src/backends/backend/triton_backend_manager.cc:1]: Remove the "(c)" to avoid this line wrap... it will cause L0_copyright to fail

File: src/backends/backend/triton_model_instance.cc [hunk 2]
```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2020-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright (c) 2020-2021, NVIDIA CORPORATION & AFFILIATES. All rights
```
Comment (deadeyegoodwin) [inline src/backends/backend/triton_model_instance.cc:1]: Remove the "(c)" to avoid this line wrap... it will cause L0_copyright to fail

**Review by deadeyegoodwin — CHANGES_REQUESTED** [review]

**Review by deadeyegoodwin — DISMISSED** [review]

**Review by deadeyegoodwin — APPROVED** [review]

**Review by deadeyegoodwin — APPROVED** [review]

File: src/core/infer_request.cc [hunk 3]
```diff
@@ -808,7 +808,7 @@ InferenceRequest::Input::SetIsShapeTensor(const bool is_shape_tensor)
 }
```
Comment (CoderHam) [inline src/core/infer_request.cc:808]: Please update copyright, and remaining source files that require updates

**Review by CoderHam — CHANGES_REQUESTED** [review]

**Review by CoderHam — APPROVED** [review]

**Review by deadeyegoodwin — APPROVED** [review]
